### ISSUE TITLE
Cannot get all Subscribers for a channel

### ISSUE THREAD

**Issue by BarryCarlyon** [twitchdev/issues#172]
**Brief description**

Using code similar to #18 but for the subscribers endpoint

I get a HTTP 500 error after getting a number of pages into downloading all subscribers for a channel

**How to reproduce**

Use code in #18 but for the subscribers endpoint
Download all the subscribers
Observe a 500 error when attempting to get the last page. Meaning the call to get all subs "fails" and you can't get all subscribers of a channel.

**Expected behavior**

Get all pages of subscribers from the helix endpoint

**Additional context or questions**

Original report on Discord

https://discordapp.com/channels/504015559252377601/523675960797691915/734923497398206566

Replicated by myself and on the forums

https://discuss.dev.twitch.tv/t/helix-subscriptions-api-returning-error-after-reading-subscriber-800-900-of-a-streamer/27248

**Comment by BarryCarlyon** [comment]
Test code.

```
const fs = require('fs');
const path = require('path');

const got = require('got');

var users = {};

let page = 0;
fetchPage = function(c) {
    console.log('Go', c);
    got({
        url: 'https://api.twitch.tv/helix/subscriptions',
        searchParams: {
            broadcaster_id: 'CASTERID',
            after: c,
            first: 100
        },
        headers: {
            authorization: 'Bearer ATOKEN'
        },
        responseType: 'json'
    })
    .then(resp => {
        page++;

        for (var x=0;x<resp.body.data.length;x++) {
            users[resp.body.data[x].user_id] = 1;
        }

        console.log('Page',page,'has',resp.body.data.length,'users',Object.keys(users).length, c);

        fs.appendFileSync(path.join(
            __dirname,
            'pages',
            page + '.json'
        ), JSON.stringify(resp.body,null,4));
        if (resp.body.pagination && resp.body.pagination.cursor) {
            return fetchPage(resp.body.pagination.cursor);
        }
    })
    .catch(err => {
        console.log(err, err.response.headers);
    });
    return;
}
fetchPage('');
```

**Comment by jbulava** [comment]
I fix for pagination of subscriptions has been deployed to fix this issue.  The pagination from #18 is a separate issue to address
